ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html

```diff
@@ -36,7 +36,7 @@
      <div data-sly-call="${longDescription.longDescription @componentId=panel.id, longDescription=panel.description, bemBlock='cmp-verticaltabs'}" data-sly-unwrap></div>
 <div class="cmp-verticaltabs__tabs-container" id="${widgetId}">
  <ol data-sly-test="${tabs.items && tabs.items.size > 0}"
-     data-sly-list.tab="${tabs.items}"
+     data-sly-list.tab="${panel.items}"
```

Comment:
same comment as given for tabs on top

Reply:
tabs.accessibilityLabel and tabs.activeItem are coming from tabs, so we still needs tabs sling model.